tests/group/test_group_external_nonposix.yml

```diff
@@ -0,0 +1,133 @@
+---
+- name: Test group
+  hosts: ipaserver
+  become: true
+  gather_facts: false
```

Comment:
I think we should not add `gather_facts: false` any more.

Reply:
fixed.